src/Maestro/maestro-angular/src/app/app.component.ts

```diff
@@ -21,7 +21,7 @@ export class AppComponent implements OnInit {
   public title = ".NET Mission Control";
   public navbarOpen = false;
   public sidebarOpen = false;
-  public returnUrl = location.pathname + location.search;
+  public returnUrl = (location.pathname + location.search).replace("http:", "https:");
```

Comment:
Looking at what the `returnUrl` is, this won't work:
https://developer.mozilla.org/en-US/docs/Web/API/Location/pathname

Reply:
yup, you're right